src/v2/guide/single-file-components.md

```diff
@@ -1,48 +1,48 @@
 
-Esto puede funcionar muy bien para proyectos de tamaño pequeño a mediano, donde JavaScript es usado solamente para mejorar algunas vistas. Sin embargo, en proyectos con mayor complejidad, or donde tu front-end es enteramente basado conducido por JavaScript, estas desventajas se hacen evidentes:
+Esto puede funcionar muy bien para proyectos pequeños y medianos, donde JavaScript solo se usa para mejorar ciertas vistas. Sin embargo, en proyectos más complejos, o cuando la interfaz está exclusivamente impulsada por JavaScript, varias desventajas se hacen evidentes:
 
-- **Definiciones globales** fuerza a nombres unicos para cada componente
-- **Plantilla de texto** carecen de resaltado de sintaxis y requieren el uso de barras para HTML multilínea
-- **Sin soporte CSS** significa que mientras el HTML y JavaScript son modularizados en componentes, el CSS es omitido
-- **Sin compilación** nos restringe a HTML y ES5 JavaScript, en lugar de preprocesadores como Pug (anteriormente Jade) y Babel
+- Las **definiciones globales** nos obligan a usar nombres únicos para cada componente
+- Las **plantillas de texto** carecen de resaltado de sintaxis y requieren feas barras para generar HTML multilínea
+- La **ausencia de soporte para CSS** significa que, en tanto HTML y JavaScript son modularizados en componentes, el CSS queda visiblemente fuera
+- **Sin una etapa de compilación** nos vemos restringidos a HTML y JavaScript ES5, en lugar de preprocesadores como Pug (anteriormente Jade) y Babel
 
-Todos estos puntos estan resueltos por **single-file components** con extensión `.vue`, lo cual es posible con herramientas como Webpack or Browserify.
+Todo esto se resuelve mediante componentes de un solo archivo con extensión .vue, hecho posible con herramientas de compilación como Webpack o Browserify.
 
 Aquí tenemos un ejemplo de un archivo que llamaremos `Hello.vue`:
 
-<a href="https://gist.github.com/chrisvfritz/e2b6a6110e0829d78fa4aedf7cf6b235" target="_blank"><img src="/images/vue-component.png" alt="Single-file component example (click for code as text)" style="display: block; margin: 30px auto;"></a>
+<a href="https://gist.github.com/chrisvfritz/e2b6a6110e0829d78fa4aedf7cf6b235" target="_blank"><img src="/images/vue-component.png" alt="Ejemplo de Componente de un Solo Archivo (click para ver código como texto)" style="display: block; margin: 30px auto;"></a>
 
 Ahora tenemos:
 
 - [Resaltado completo de la sintaxis](https://github.com/vuejs/awesome-vue#source-code-editing)
-- [Módulos CommonJSCommonJS modules](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
-- [Definición de alcance del CSS por Component](https://vue-loader.vuejs.org/en/features/scoped-css.html)
+- [Módulos CommonJS](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
+- [CSS con ámbito de componente](https://vue-loader.vuejs.org/en/features/scoped-css.html)
```

Comment:
What do you think about something like `CSS con alcance de componente`? I think we shouldn't lose the meaning of scoped 🤔

Reply:
I searched the web for generally accepted translations, there doesn't seem to be an agreement. I agree alcance could be a better term in this context.